student-test/assignment1/Test_validate_username.c

```diff
@@ -15,8 +15,12 @@
 void test_validate_my_username()
 {
     /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
+     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char *username_validate = my_username();
+    char *username_conf = malloc_username_from_conf_file();
+
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(username_validate, username_conf, "Success!");
```

Comment:
@DeekshithPatil purpose of the message here is to report an error on username mismatch.